modules/ROOT/pages/anypoint-custom-metrics-connector.adoc

```diff
@@ -6,11 +6,11 @@ endif::[]
 
 The Anypoint Custom Metrics connector enables you to send operational and business metric values from a hybrid application to Anypoint Monitoring, where the metrics can be visualized in real time using the built-in *Custom metrics* dashboard. 
 
-Once you select the metrics to display in the dashboard, you can filter and group the metrics along the included dimensions. You can include multiple custom and built-in metrics in a single dashboard visualization.
+Once you select the metrics you defined in the Custom Metrics connector to display in the dashboard, you can filter and group the metrics along the included dimensions. You can include multiple custom and built-in metrics in a single dashboard visualization.
 
 [NOTE] 
 ====
-To use the Anypoint Custom Metrics connector, you must have a Titanium subscription. +
+To see metrics that you configured in the Anypoint Custom Metrics connector in Anypoint Monitoring, you must have a Titanium subscription. +
```

Comment:
Or, To see metrics in Anypoint Monitoring, you must have a T sub and must configure the metrics in ACM...